protocols/peer/test/Peer.js

```diff
@@ -3,17 +3,27 @@ const Swap = artifacts.require('Swap')
 const Types = artifacts.require('Types')
 const FungibleToken = artifacts.require('FungibleToken')
 
-const { emitted, reverted, equal } = require('@airswap/test-utils').assert
-const { takeSnapshot, revertToSnapShot } = require('@airswap/test-utils').time
+const { emitted, reverted, equal, ok } = require('@airswap/test-utils').assert
+const { balances } = require('@airswap/test-utils').balances
+const {
+  takeSnapshot,
+  revertToSnapShot,
+  getTimestampPlusDays,
+  advanceTime,
+} = require('@airswap/test-utils').time
 const { orders } = require('@airswap/order-utils')
+const {
+  EMPTY_ADDRESS,
+  SECONDS_IN_DAY,
+} = require('@airswap/order-utils').constants
 
 let snapshotId
 
 contract('Peer', async accounts => {
-  let aliceAddress = accounts[0]
-  let bobAddress = accounts[1]
-  let carolAddress = accounts[2]
-  let davidAddress = accounts[3]
+  let aliceAddress = accounts[1]
+  let bobAddress = accounts[2]
+  let carolAddress = accounts[3]
+  let aliceTradeWallet = accounts[5]
```

Comment:
any reason for skipping 4?

Reply:
oh david was after carol and then I realised he wasn't being used. Will update the trade wallet to 4.